lib/dstu2_resources.rb

```diff
@@ -1,6 +1,319 @@
 module Cerner
   module Resources
 
+    DSTU2_MEDICATION_STATEMENT_BUNDLE ||= {
+      "resourceType"=> "Bundle",
+      "id"=> "b8ea9905-29fe-4d40-9d71-475ca2ebb209",
+      "type"=> "searchset",
+      "total"=> 3,
+      "link"=> [
+        {
+          "relation"=> "self",
+          "url"=> "https://fhir-open.sandboxcernerpowerchart.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/MedicationStatement?patient=1316024&_count=50"
+        }
+      ],
+      "entry"=> [
+        {
+          "fullUrl"=> "https://fhir-open.sandboxcernerpowerchart.com/dstu2/d075cf8b-3261-481d-97e5-ba6c48d3b41f/MedicationStatement/16358117",
+          "resource"=> {
+            "resourceType"=> "MedicationStatement",
+            "id"=> "16358117",
+            "meta"=> {
+              "versionId"=> "1"
+            },
+            "text"=> {
+              "status"=> "generated",
+              "div"=> "&lt;div>&lt;p>&lt;b>Medication Statement&lt;/b>&lt;/p>&lt;p>&lt;b>Patient Name&lt;/b>: PETERS, TIMOTHY&lt;/p>&lt;p>&lt;b>Medication Name&lt;/b>: aspirin&lt;/p>&lt;p>&lt;b>Dosage Instructions&lt;/b>: 325 mg, Oral, q4hr, PRN: headache&lt;/p>&lt;p>&lt;b>Status&lt;/b>: Active&lt;/p>&lt;/div>"
+            },
+            "contained"=> [
+              {
+                "resourceType"=> "Medication",
+                "id"=> "2750067",
+                "code"=> {
+                  "coding"=> [
+                    {
+                      "system"=> "http://www.nlm.nih.gov/research/umls/rxnorm",
+                      "code"=> "1191",
+                      "display"=> "Aspirin",
+                      "userSelected"=> false
+                    }
+                  ],
+                  "text"=> "aspirin"
+                },
+                "product"=> {
+                  "form"=> {
+                    "coding"=> [
+                      {
+                        "system"=> "http://snomed.info/sct",
+                        "code"=> "385055001",
+                        "display"=> "Tablet dose form (qualifier value)",
+                        "userSelected"=> false
+                      }
+                    ],
+                    "text"=> "Tab"
+                  }
+                }
+              }
+            ],
+            "patient"=> {
+              "reference"=> "Patient/1316024",
+              "display"=> "PETERS, TIMOTHY"
```

Comment:
We were changing this on other resources so that we don't plaster Tim's name on our site.